distributions/otelcol/manifest.yaml

```diff
@@ -7,61 +7,45 @@ dist:
   otelcol_version: 0.34.0
 
 receivers:
-  - gomod: github.com/open-telemetry/opentelemetry-collector-contrib/receiver/hostmetricsreceiver 9b0e90027913f53d4231eb1762063cf67df61051
-  - gomod: github.com/open-telemetry/opentelemetry-collector-contrib/receiver/jaegerreceiver 9b0e90027913f53d4231eb1762063cf67df61051
-  - gomod: github.com/open-telemetry/opentelemetry-collector-contrib/receiver/kafkareceiver 9b0e90027913f53d4231eb1762063cf67df61051
-  - gomod: github.com/open-telemetry/opentelemetry-collector-contrib/receiver/opencensusreceiver 9b0e90027913f53d4231eb1762063cf67df61051
-  - gomod: github.com/open-telemetry/opentelemetry-collector-contrib/receiver/prometheusreceiver 9b0e90027913f53d4231eb1762063cf67df61051
-  - gomod: github.com/open-telemetry/opentelemetry-collector-contrib/receiver/zipkinreceiver 9b0e90027913f53d4231eb1762063cf67df61051
-  - import: go.opentelemetry.io/collector/receiver/otlpreceiver
```

Comment:
Components that are part of the core should be specified only if the distribution has a "include_core: false" option. Otherwise, all core components are being included. Adding it here as well causes runtime errors like this:

```
2021/09/02 20:49:31 failed to build components: [duplicate extension factory "zpages"; duplicate receiver factory "otlp"; duplicate exporter factory "logging"; duplicate processor factory "batch"]
```